file-manager/src/main/scala/it/pagopa/interop/commons/files/service/impl/S3ManagerImpl.scala

```diff
@@ -162,4 +166,23 @@ final class S3ManagerImpl(blockingExecutionContext: ExecutionContextExecutor)(
   def calcContentMd5(file: File): String = calcContentMd5(Files.readAllBytes(file.toPath))
 
   def calcContentMd5(byteArray: Array[Byte]): String = new String(Base64.encodeBase64(DigestUtils.md5(byteArray)))
+
+  override def generatePresignedUrl(containerPath: String, path: String): Try[String] = {
```

Comment:
this implementation is ok, but it creates a presigned url only for retrieving an object. It is still required, but for this specific ticket you need an presigned url to store an object.
See [this](https://docs.aws.amazon.com/sdk-for-java/latest/developer-guide/examples-s3-presign.html#put-presignedobject) doc.

I suggest to rename the function to make it explicit
```suggestion
  override def generateGetPresignedUrl(containerPath: String, path: String): Try[String] = {
```